server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/jsieve/RedirectAction.java

```diff
@@ -61,7 +61,6 @@ public void execute(ActionRedirect anAction, Mail aMail, ActionContext context)
         recipients.add(new MailAddress(new InternetAddress(anAction.getAddress())));
         MailAddress sender = aMail.getSender();
         context.post(sender, recipients, aMail.getMessage());
-        aMail.setState(Mail.GHOST);
```

Comment:
Idem, don't ghost the mail for every recipient, because one people executed a SIEVE script.